lib/core/src/display/shape/text.rs

```diff
@@ -125,8 +131,7 @@ impl TextComponent {
     }
 
     fn refresh_cursors(&mut self, fonts:&mut Fonts) {
-        let cursors_changed = !self.cursors.dirty_cursors.is_empty();
-        if cursors_changed {
+        if self.cursors.dirty {
             let gl_context = &self.gl_context;
             let content    = &mut self.content;
```

Comment:
I'm happy this could could be obsolete after our integration. It's tricky to handle it manually like here.